package.json

```diff
@@ -2,21 +2,24 @@
   "name": "package.land",
   "private": true,
   "description": "A nice place for JavaScript packages",
-  "main": "index.js",
+  "main": "server.js",
   "repository": "https://github.com/nice-registry/package.land",
   "author": "zeke",
   "license": "MIT",
   "scripts": {
     "start": "node server.js",
-    "test": "mocha && standard --fix && standard-markdown"
+    "test": "NODE_ENV=test mocha && standard --fix && standard-markdown",
+    "dev": "nodemon"
   },
   "dependencies": {
     "all-the-package-repos": "^1.2.1",
-    "express": "^4.16.2"
+    "express": "^4.16.2",
+    "lil-env-thing": "^1.0.0"
   },
   "devDependencies": {
     "chai": "^4.1.2",
     "mocha": "^4.0.1",
+    "nodemon": "^1.12.1",
```

Comment:
I want to suggest you use [svr](https://github.com/Kikobeats/svr) instead. While nodemon reload ALL the code, svr only refresh the modified files from require memory cache. The results is that is faster and less boring.

It's inspired in [micro-dev](https://github.com/zeit/micro-dev), but for express-like http servers